README.md

```diff
@@ -1,29 +1,267 @@
 
-## Acknowledgements
+Some of our key features are:
 
-This project was initialized with [create-t3-app](https://github.com/t3-oss/create-t3-app)
+  <ol>
+    <li>
+      Secure Sign-in:  Rest easy knowing that your pet's safety and privacy are our priorities too!
+    </li>
+    <li>
+      Messaging Platform: Stay connected with fellow pet lovers through our intiuative messaging system.
+    </li>
+    <li>
+      Forums: Dive into dedicated forums tailored to your pet's specific needs and characteristics.
+    </li>
+    <li>
+      Friend-Finding: Discover new friends for both you and your pet! Utilize our advanced friend-finding features to connect with local pet owners! 
+    </li>
+  </ol>
+</p>
 
+### Built With
 
+<!-- TODO Add proper colors for each badge -->
+
+![Typescript](https://img.shields.io/badge/Typescript-%23000000.svg?style=for-the-badge&logo=Typescript)
+![next.js](https://img.shields.io/badge/next-%23000000.svg?style=for-the-badge&logo=next.js)
+![tRPC](https://img.shields.io/badge/tRPC-%23000000.svg?style=for-the-badge&logo=tRPC)
+![React](https://img.shields.io/badge/React-%23000000.svg?style=for-the-badge&logo=React)
+<br />
+![Prisma](https://img.shields.io/badge/Prisma-%23000000.svg?style=for-the-badge&logo=Prisma)
+![Supabase](https://img.shields.io/badge/Supabase-%23000000.svg?style=for-the-badge&logo=Supabase)
+![TailwindCSS](https://img.shields.io/badge/tailwindcss-%230000.svg?style=for-the-badge&logo=tailwind-css)
+![DaisyUI](https://img.shields.io/badge/DaisyUI-%23000000.svg?style=for-the-badge&logo=DaisyUI)
+
+<p align="right">
+  (<a href="#readme-top">back to top</a>)
+</p>
+
+## Getting Started
+
+<p>
+  Before getting started with PetCentral please ensure that you have a database set up to work with Prisma. We highly recommend <a href="https://supabase.com" target="_blank">Supabase</a>
+</p>
+
+### Prerequisites
+
+![NPM](https://img.shields.io/badge/NPM-%23000000.svg?style=for-the-badge&logo=npm&logoColor=white)
+
+```sh
+npm install npm@latest -g
+```
+
+![Node.js](https://img.shields.io/badge/Node.js-%23000000.svg?style=for-the-badge&logo=node.js)
+
+```sh
+nvm install --lts
+node --version
+```
+
+Node version is expected to be v20.10.0 or higher
+
+### Installation
+
+1. Clone the repo
+   ```sh
+   git clone https://github.com/SA-BlueOcean/PetCentral.git
+   ```
+1. Install NPM packages
+   ```sh
+   npm install
+   ```
+1. Rename `.env.example` to `.env` and fill in the following fields:
+   - `DATABASE_URL`
+   - `NEXTAUTH_URL`
+   - `NEXTAUTH_SECRET` (use `# openssl rand -base64 32` to generate)
+   - `GITHUB_CLIENT_ID`
+   - `GITHUB_CLIENT_SECRET`
+   - `GOOGLE_CLIENT_ID`
+   - `GOOGLE_CLIENT_SECRET`
+   - `FACEBOOK_CLIENT_ID`
+   - `FACEBOOK_CLIENT_SECRET`
+   - Check NextAuth.js documentation for more information on how to set up OAuth providers
+1. Run Prisma generate to generate the Prisma client
+   ```sh
+   npm run prisma generate
```

Comment:
this should be `npx prisma generate` or `npm run postinstall`

Reply:
Fixed the command in the readme to `npx prisma generate` 

Great idea on the .env.example additions 
I completely forgot to adjust it to reflect changes. 
Updated to include all required env variables.